c2rust-analyze/tests/common/mod.rs

```diff
@@ -21,6 +22,20 @@ impl Analyze {
 
         let manifest_path = dir.join("Cargo.toml");
         let rs_path = dir.join(rs_path); // allow relative paths, or override with an absolute path
+
+        let directives = fs::read_to_string(&rs_path)
+            .unwrap()
+            .split('\n')
+            .flat_map(|line| {
+                line.trim()
+                    .strip_prefix("//!")
```

Comment:
Is there a reason to use the doc-comment prefix `//!` for these directives?  Maybe something unused like `//*` or `// FLAGS:` would be better.

Reply:
That's what the directives for transpilation tests in `tests/` used so I just copied them.